### ISSUE TITLE
[suggestion] silver to diamond upgrade item

### ISSUE THREAD

**Issue by blockgoblin31** [progwml6/ironchest#283]
Add a upgrade for silver chest to diamond chest. This is the only chest upgrade without an equivalent item.

**Comment by samtaylorr** [comment]
This is possible correct? There is an upgrade from sliver chests to gold chests, and from there you can upgrade to diamond.

**Comment by blockgoblin31** [comment]
yes, but there is a crafting recipe where silver chest + 5 glass + 3 diamonds = diamond chest. This is the only chest upgrade recipe without it's in world counterpart

**Comment by blockgoblin31** [comment]
basically, you save 4 gold and 1 step if you break the chest to upgrade it
